op/model.py

```diff
@@ -47,6 +48,17 @@ def get_relation(self, relation_name, relation_id=None):
     def get_unit(self, unit_name):
         return self._cache.get(Unit, unit_name)
 
+    @property
+    def pod_spec(self):
```

Comment:
The "pod" is a central concept in Kubernetes, which means we'll probably regret this organization once we need to do anything else about it than getting and setting the spec.